<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>World</class>
 <widget class="QWidget" name="World">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>99</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>99</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>99</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>World &amp; Seed</string>
  </property>
  <widget class="QSpinBox" name="s_seed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-999999999</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_year">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="suffix">
    <string> EY</string>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_day">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="prefix">
    <string>Day </string>
   </property>
   <property name="maximum">
    <number>365</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_hour">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="prefix">
    <string>Hour </string>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="p_end">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>End Times</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
